_episodes/l1-02-branching-merging.md

```diff
@@ -84,24 +84,18 @@ interface" or "fixing bug in matrix inversion algorithm".
 
 > ## Starting point
 >
-> This episode will take as starting point for further development [a zipped repository
-> containing the recipe for guacamole sauce](../code/recipe_with_history.zip).
+> Navigate to your `recipe` directory, containing the guacamole recipe repository
+> developed in the [introductory course]().
```

Comment:
URL is missing here, but we probably don't need to reference the introductory course so much as the Setup for this course